src/db/Dockerfile

```diff
@@ -0,0 +1,53 @@
+FROM python:3.10-bullseye as build
+
+ARG OUT_DIR=/tmp/ugbio
+RUN mkdir -p $OUT_DIR
+
+COPY ./src/ ./src/
+
+#build ugbio_core
+RUN python -m pip install build
+RUN python -m build --outdir $OUT_DIR --wheel ./src/core
```

Comment:
You don't need core in your docker. Instead you are missing your package code. See other dockerfiles in the repo as a reference and remove all packages you don't need.

Reply:
Fixed